crates/weaver_semconv/src/v2/entity.rs

```diff
@@ -7,15 +7,15 @@ use serde::{Deserialize, Serialize};
 
 use crate::{
     group::GroupSpec,
-    v2::{attribute::AttributeRef, CommonFields},
+    v2::{attribute::AttributeRef, signal_id::SignalId, CommonFields},
 };
 
 /// Defines a new entity.
 #[derive(Serialize, Deserialize, Debug, Clone, JsonSchema)]
 #[serde(deny_unknown_fields)]
 pub struct Entity {
     /// The type of the Entity.
-    pub r#type: String,
+    pub r#type: SignalId,
```

Comment:
Is an entity a signal?

Reply:
That's the (eventual) plan, yes. It's like a meta-signal now.